src/commons/error.rs

```diff
@@ -273,6 +274,7 @@ pub enum Error {
     KeyUseNoOldKey,
     KeyUseNoIssuedCert,
     KeyUseNoMatch(KeyIdentifier),
+    KeyRollNotAllowed,
```

Comment:
Observation: as `fn status()` will map this to HTTP 400 Bad Request - is that the correct mapping for this error?